dp-identity-data-migration/userdataextraction/user_extraction.go

```diff
@@ -144,10 +170,54 @@ func process_zebedee_users(csvwriter *csv.Writer, userlist []zebedee.User) {
 		csvline.mfa_enabled = "FALSE"
 		csvline.phone_number_verified = "FALSE"
 		csvline.email_verified = "TRUE"
-		if err := csvwriter.Write(convert_to_slice(csvline)); err != nil {
-			fmt.Println("error writing record to csv:", err)
+
+		if isValidEmail := validateEmailId(validEmailDomains, user.Email); isValidEmail {
```

Comment:
Do we need a variable?

```suggestion
		if validateEmailId(validEmailDomains, user.Email) {
```